lib/dolly/document.rb

```diff
@@ -1,9 +1,13 @@
 require "dolly/query"
 require "dolly/property"
+require 'dolly/timestamps'
+require 'active_model/callbacks'
 
 module Dolly
   class Document
     extend Dolly::Connection
+    extend ActiveModel::Callbacks
```

Comment:
I like @javierg 's idea. It is less magic, which is a good thing. You could even go as far as telling it which fields are timestamps, with some sensible defaults.

`timestamps! created: :foo, updated: :bar # default is :created_at, :updated_at`